pom.xml

```diff
@@ -171,7 +171,7 @@
         <junit.version>4.11</junit.version>
         <jopt-simple.version>3.2</jopt-simple.version>
         <commons-compress.version>1.5</commons-compress.version>
-        <hadoop.version>[2.0.2-alpha,2.3.0]</hadoop.version>
+        <hadoop.version>2.7.2</hadoop.version>
         <hadoop20.output.dir>target/hadoop20-classes</hadoop20.output.dir>
```

Comment:
is there any reason to keep this property at all?  seems better to just have it defined in each profile as necessary.